packages/plugin-ext/doc/how-to-add-new-custom-plugin-api.md

```diff
@@ -30,7 +36,9 @@ export class FooExtPluginApiProvider implements ExtPluginApiProvider {
                 initFunction: 'fooInitializationFunction',
                 initVariable: 'foo_global_variable'
             },
-            backendInitPath: path.join(__dirname, 'foo-init')
+            backendInitPath: path.join(__dirname, 'foo-init'),
+            // (Optional) provide the same API to headless plugins, too (or a different/subset API)
```

Comment:
Not sure I want to have the `(Optional)` here as it seems that otherwise you need to provide the other two when in fact all of them are actually optional.